artifacts/deploy/karmada-controller-manager.yaml

```diff
@@ -26,6 +26,8 @@ spec:
           command:
             - /bin/karmada-controller-manager
             - --kubeconfig=/etc/kubeconfig
+            - --bind-address=0.0.0.0
```

Comment:
Actually, this `--bind-address` has been deprecated. 
But no big deal, it will be removed once we in release-1.12.